lib/shell/common.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
Rename from `common.sh` to something more descriptive...maybe `process_control.sh` or something like that.

`common.sh` doesn't tell me what this code does.

Reply:
Renamed already. Thanks.